anvil.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+
+PORT=8545
+HOST="0.0.0.0"
+SLEEP_DURATION=0.1
+
+# Start the Anvil server.
+anvil --host "$HOST" --rpc-url "$MAINNET_RPC_URL" &
```

Comment:
I have very little understanding of how chain forking is implemented in Anvil, but we may want to consider persisting state to a file that is mounted in a volume (`--state /var/lib/anvil/state`) so that we don't incur the cost of re-syncing that state on each startup.